csvcubed/csvcubed/utils/qb/components.py

```diff
@@ -0,0 +1,41 @@
+"""
+Cube Components
+---------------
+
+Utilities to help when handling qube components data.
+"""
+
+from enum import Enum
+
+
+class ComponentPropertyType(Enum):
+    """
+    The type of qube component.
+    """
+
+    Dimension = "DimensionProperty"
+    """ The component is of type qb:Diemension. """
+
+    Attribute = "AttributeProperty"
+    """ The component is of type qb:Attribute. """
+
+    Measure = "MeasureProperty"
+    """ The component is of type qb:Measure. """
+
+
+def get_printable_component_property_type(property_type: str) -> str:
+    """
+    Produces the user-friendly name of component property type.
+
+    Member of :file:`./utils/qb/components.py`
+
+    :return: `str` - user-friendly name of component property type.
+    """
+    if ComponentPropertyType.Dimension.value in property_type:
```

Comment:
Can we check the whole URI matches exactly? This feels a bit loose using `in`.

Reply:
@robons Before I was thinking that the URLs could change. However, I've found that this is not the case according to linked-data docs. So I've updated this to URI matching.